jaxrs-api/src/main/java/javax/ws/rs/NameBinding.java

```diff
@@ -25,7 +25,7 @@
 /**
  * Meta-annotation used to create name binding annotations for filters
  * and interceptors.
- * <p>
```

Comment:
After removing this `<p>`, the upper sentence isn't in its own paragraph any more. I don't think that this is correct. The rendered page looks weird, because there is no spacing between the first paragraph and the source code any more.

Reply:
I will merge now but ask some rookie volunteer to improve the layout of the HTML.